### PR TITLE
Add Reason for Care Eligible formula

### PR DESCRIPTION
**Pull request by tolaouk**

### CODE REVIEW

File: openfisca_us/variables/hhs/ccdf.py [hunk 1]
```diff
@@ -92,12 +92,31 @@ def formula(spm_unit, period, parameters):
         return income_to_smi_ratio <= p_ratio_limit
 
 
+class meets_ccdf_activity_test(Variable):
+    value_type = bool
+    entity = SPMUnit
+    definition_period = YEAR
+    label = u"Activity test for CCDF"
+
```
Comment (nikhilwoodruff) [inline openfisca_us/variables/hhs/ccdf.py:100]: Could you add a documentation field here explaining briefly the activity test?

File: openfisca_us/variables/hhs/ccdf.py [hunk 2]
```diff
@@ -92,12 +92,31 @@ def formula(spm_unit, period, parameters):
...
+
 class is_ccdf_reason_for_care_eligible(Variable):
     value_type = bool
     entity = Person
     definition_period = YEAR
     label = u"Reason-for-care eligibility for CCDF"
```
Comment (nikhilwoodruff) [inline openfisca_us/variables/hhs/ccdf.py:107]: And here, for the reason-for-care eligibility test.

**Review by nikhilwoodruff — APPROVED** [review]
Thanks @tolaouk, looks great! Just a small metadata-related comment (keeping the checklist in the PR template might help catch this in future). But apart from that, looks good to merge after these suggestions.

File: openfisca_us/variables/demographic/person.py [hunk 3]
```diff
@@ -120,3 +120,10 @@ class is_surviving_child_of_disabled_veteran(Variable):
         "Indicates whether a person is a surviving child of a disabled veteran"
     )
     label = "Surviving child of disabled veteran"
+
+
+class is_receiving_or_needs_protective_services(Variable):
```
Comment (MaxGhenis) [inline openfisca_us/variables/demographic/person.py:125]:
```suggestion
class receives_or_needs_protective_services(Variable):
```

File: openfisca_us/variables/hhs/ccdf.py [hunk 4]
```diff
@@ -92,12 +92,33 @@ def formula(spm_unit, period, parameters):
...
+    definition_period = YEAR
+    documentation = "Indicates whether parent or parents meet activity test (working/in job training/in educational program)"
+    label = u"Activity test for CCDF"
+
+
 class is_ccdf_reason_for_care_eligible(Variable):
     value_type = bool
     entity = Person
     definition_period = YEAR
+    documentation = "Indicates whether child qualifies for ccdf based on parents meeting activity test or that he/she receives or needs protective services"
```
Comment (MaxGhenis) [inline openfisca_us/variables/hhs/ccdf.py:107]:
```suggestion
    documentation = "Indicates whether child qualifies for CCDF based on parents meeting activity test or that he/she receives or needs protective services"
```

**Pull request merged**